### PR TITLE
Fix/3/manual refactor-Review

### LINKED ISSUE #3: Format-nonstandardcode
Create new branch(fix/issue-id/manual-refactor) from v0.1
Reformat code using black ,isort
Run flake8 to fix any errors identified

### PR DESCRIPTION
**Pull request by Vidhya-Sagar-Reddy**
1.Created new-branch fix/3/manual-refactor to fix the issue#3
2.Added formatted code and flake8 configuration.
image

### CODE REVIEW

**Comment by prateekta** [comment]
since you've already created a tag, you don't need to create a new file called nonstandardcode-formatted.py. You can update the existing file.

**Comment by Vidhya-Sagar-Reddy** [comment]
> since you've already created a tag, you don't need to create a new file called nonstandardcode-formatted.py. You can update the existing file.

**Review by prateekta — DISMISSED** [review]
looks good. you can merge the changes

File: README.md [hunk 1]
```diff
@@ -18,4 +18,9 @@ The following techniques have been used:
   conda env create -f env.yml
   
 ## To excute the script
+<<<<<<< HEAD
```
Comment (prateekta) [inline README.md:21]: you've merged incorrectly. We can talk about this during our meeting

**Pull request merged**